requirements/mypy.txt

```diff
@@ -11,3 +11,4 @@ types-PyYAML==6.0.12.2
 types-requests==2.28.11.5
 types-six==1.16.21.4
 types-urllib3==1.26.25.4
+types-python-dateutil
```

Comment:
`mypy` forced me to add ths. I think we can agree that we don't need to specify a version on these, right ...?